pyproject.toml

```diff
@@ -76,7 +76,7 @@ dependencies = [
   "grpcio-tools<1.71",
   "protobuf<6.31",
   "pycryptodome<3.23",
-  "sqlalchemy-cratedb<0.42",
+  "sqlalchemy-cratedb==0.42.0.dev0",
```

Comment:
It will be good to ship development releases with `pre-alpha` releases of downstream artefacts. After relevant upstream patches from https://github.com/crate/sqlalchemy-cratedb/pull/215 have been slotted in, and a GA release has been made, the version will be bumped.